### PR TITLE
Update BQ Lens FAQ and editorial changes

### PR DESCRIPTION
**Pull request by azure562**
- this commit is for collaboration purpose

### CODE REVIEW

File: website/docs/dashboards/bigquery-lens.mdx [hunk 1]
```diff
@@ -5,171 +5,145 @@ description: Your Swiss army knife for highlighting inefficiencies in your organ
...
-
-## Setup
+## Set up BigQuery Lens
 
-Once the Google Cloud Service Account setup is complete, select the _Attach_ button from the dashboard navigation bar, then select _BigQuery Lens_ from the list of options.
+To access the BigQuery Lens, make sure that you have followed the steps at [Connect your Google Cloud Organization](../google-cloud/connect-google-cloud-service-account.mdx) to upload your Google Cloud service account and grant the appropriate permissions to the CMP.
 
-After attaching the BigQuery Lens dashboard, a BigQuery Lens icon will appear on your dashboard and the CMP will start gathering information about your account usage patterns.
+1. To check the status of your service account, select the _Settings_ icon (a gear) from the top menu bar, then select **Google Cloud**.
+2. Select the BigQuery Lens feature on the screen to enable it. The same is true for BigQuery Lens Advanced.
```
Comment (norosa) [inline website/docs/dashboards/bigquery-lens.mdx:15]: It's best to separate each list item with an empty line

File: website/docs/dashboards/bigquery-lens.mdx [hunk 2]
```diff
@@ -5,171 +5,145 @@ description: Your Swiss army knife for highlighting inefficiencies in your organ
...
+## Dashboard
 
-:::
+The BigQuery Lens dashboard consists of several widgets that highlight different aspects of your BigQuery usage.
 
-## Understanding the BigQuery Lens
+Before diving into the widgets, have a look at the options provided in the two dropdown lists above the widgets area:
 
-The BigQuery Lens dashboard is made up of widgets that, each designed to highlight a different aspect of your BigQuery usage:
+- **Time range**: Adjusts the time frame of the data you're looking at. Options: `Last 30 days`, `Last 7 days`, and `Last day`.
```
Comment (norosa) [inline website/docs/dashboards/bigquery-lens.mdx:38]: I'm not sure that literal syntax is appropriate here. Could we instead italicize the phrases, like _Last 30 days_. Or else, put them in quotes?

File: website/docs/dashboards/bigquery-lens.mdx [hunk 3]
```diff
@@ -5,171 +5,145 @@ description: Your Swiss army knife for highlighting inefficiencies in your organ
...
 
-1. _Time range and Usage type dropdowns_:
+### Explorer widget
 
-   - _Time range_ &mdash; Adjusts the time frame of the data you're looking at, and has options of _Last 30 days_, _Last 7 days_, or _Last day_
-   - _Usage type_ &mdash; Determines which BigQuery projects you're looking at, based on the pricing model you're using to pay for them, and has options of _On demand_ or _Flat-rate_
+Get more granular information as far as your team's BigQuery usage, adjustable by the user in the following format:
 
-2. _Recommendations_ &mdash; Comprehensive recommendations with details about how to act on each suggestion.
+- `On-Demand` usage type: <!-- textlint-disable max-comma -->(_Project_, _Billing Project_, _Dataset_, _Table_, _User_)<!-- textlint-enable --> by (_Scan Price_, _Scan TB_)
```
Comment (norosa) [inline website/docs/dashboards/bigquery-lens.mdx:54]: Same comment here re the use of literals.

File: website/docs/dashboards/bigquery-lens.mdx [hunk 4]
```diff
@@ -5,171 +5,145 @@ description: Your Swiss army knife for highlighting inefficiencies in your organ
...
-- _Cluster your tables_ &mdash; Cluster the tables listed under the _Table_ column by the fields suggested under the _Cluster By_ column.
-- _Enforce Partition Fields_ &mdash; Use the suggested partitioned fields under the _Partition Field_ column for the corresponding queries listed under the _Query ID_ column.
-- _Partition your tables_ &mdash; Partition the tables listed under the _Table_ column by the suggested fields listed under the _Partition Fields_ column.
-- _Limit query jobs_ &mdash; Reduce execution frequency of the listed jobs under the _Query ID_ column by the percentage you choose on the slider and view the associated savings of each reduced job under the _Savings by Reducing Jobs_ column.
-- _Switch to monthly flat-rate plan_ &mdash; Purchase the slots amount listed in the _Optimal slots amount_ column to reduce your BigQuery spend.
+### On-demand recommendations
 
-#### Flat-rate recommendations
+- **Backup and remove unused tables**: Back up and remove the tables listed under the **Table** column. If the table has multiple partitions, select the number under **Partition(s) to Remove** to see which partitions should be removed.
```
Comment (norosa) [inline website/docs/dashboards/bigquery-lens.mdx:81]: Same comment re list item spacing.

File: website/docs/dashboards/bigquery-lens.mdx [hunk 5]
```diff
@@ -5,171 +5,145 @@ description: Your Swiss army knife for highlighting inefficiencies in your organ
...
+- **Switch to monthly flat-rate plan**: Purchase the slots amount listed in the **Optimal slots amount** column.
 
-- _Move repeating queries to a different time slot_ &mdash; Lower your minimum average slots requirements by moving the recurring queries to a less-busy hour of the day.
+### Flat-rate recommendations
 
-  Review the _BigQuery Slots Consumption_ widget with the interval set to _Hour_ to identify suitable hours. Any troughs in your daily slots consumption indicate optimal hours to run recurring queries.
-
-- _Backup and remove unused tables_ &mdash; Back up and remove the unused tables to lower costs.
+- **Move repeating queries to a different time slot**: Lower your minimum average slots requirements by moving the recurring queries to a less-busy hour of the day. Review the [Slots Consumption widget](#slots-consumption-widget) with the interval set to `Hour` to identify suitable hours. Troughs in your daily slots consumption indicate optimal hours to run recurring queries.
```
Comment (norosa) [inline website/docs/dashboards/bigquery-lens.mdx:90]: List item spacing.

**Review by norosa — COMMENTED** [review]
I have only reviewed the markup, not the content. Will leave that to @mtagda. :)

**Comment by mtagda** [comment]
@azure562 regarding "What queries do you run":
Here only the following part is true
```
Daily run of cost simulation queries to provide the cost savings information and potential savings.
Daily run of lens calculation queries to provide detailed BigQuery usage.
```
We don't have the enrichment process anymore. Instead, we have a **tables discovery process** which runs on a schedule, collects tables metadata and writes it into the table called `tables` which we create in the `doitintl-cmp-bq` dataset. Wondering if it should maybe go into a separate section? I'll leave it with you :) 
So, to sum up, we have 2 processes: daily dashboard update (simulation/calculation queries mentioned above) and tables discovery process. Let me know if this is clear or you need more details 🙏

**Comment by mtagda** [comment]
@azure562 we need another update for "How much does BigQuery Lens cost?"
I think we can leave the "usually it's $10 per month" but we need to make sure to mention that it really depends on the volume of logs. What I want to say is that if a customer is a heavy BQ user the cost will be much higher since they pay for scan (unless they use reservations) and we scan the logs several times every day as part of the process.

File: website/docs/dashboards/bigquery-lens.mdx (hunk 4 above)
Comment (azure562) [reply website/docs/dashboards/bigquery-lens.mdx:81]: An empty line will generate an addtional pair of <p>...</p> tags and increase the spacing between adjacent line items. I feel we can choose whether to add that line per our own judgment.

File: website/docs/dashboards/bigquery-lens.mdx (hunk 1 above)
Comment (azure562) [reply website/docs/dashboards/bigquery-lens.mdx:15]: An empty line will generate an additional pair of `<p></p>` tags and increase the spacing between adjacent line items. I feel we can choose whether to add that line per our own judgment.

File: website/docs/dashboards/bigquery-lens.mdx [hunk 6]
```diff
@@ -184,19 +157,23 @@ We may not be able to detect which table is being scanned because of:
   alt="A screenshot highlighting the _Not yet allocated_ figure"
 />
 
-### How much does BigQuery Lens cost monthly?
+### How much does BigQuery Lens cost?
+
+BigQuery Lens creates and maintains datasets on your behalf and runs queries to provide you with analytics data. This process generally costs about $10 per month.
```
Comment (mtagda) [inline website/docs/dashboards/bigquery-lens.mdx:162]: "creates and maintains datasets" sounds a little bit incorrect, we only create 1 dataset, 1 sink and 1 extra table 🤔  The cost is for running queries only (if we want to be very exact then there is also some storage cost involved but this should be a very small number)

Comment (azure562) [reply website/docs/dashboards/bigquery-lens.mdx]: updated at https://github.com/doitintl/help/pull/104/commits/eca8fec66ec484d7f3922040b97e7b6b53f3e807

BigQuery Lens creates and maintains the `doitintl-cmp-bq` dataset on your behalf and runs queries to provide you with analytics data. The cost for running the queries is usually around $10 per month.

File: website/docs/dashboards/bigquery-lens.mdx [hunk 7]
```diff
@@ -5,171 +5,144 @@ description: Your Swiss army knife for highlighting inefficiencies in your organ
...
 
 ### What sinks do you create?
 
-A sink for query jobs is created in the billing project attached to the service account you add to the CMP. This sink pushes all your BigQuery jobs into a table under `<PROJECTID>.doitintl-cmp-bq.cloudaudit_googleapis_com_data_access`.
+A sink for query jobs is created in the billing project attached to the service account you added to the CMP. This sink pushes all your BigQuery jobs into a table under `<PROJECTID>.doitintl-cmp-bq.cloudaudit_googleapis_com_data_access`.
 
 ### What queries do you run?
 
-We have two main processes, one of which runs queries and the other which runs the enrichment process:
+BigQuery Lens has two main processes: a table discovery process to prepare the data, and a dashboard update process to run queries.
```
[the remark this answers is not in the document]
Comment (azure562) [reply website/docs/dashboards/bigquery-lens.mdx]: updated at https://github.com/doitintl/help/commit/eca8fec66ec484d7f3922040b97e7b6b53f3e807, pls check again :)

**Review by mtagda — APPROVED** [review]
